concordium-cis2/src/lib.rs

```diff
@@ -62,30 +62,6 @@ pub const UPDATE_OPERATOR_EVENT_TAG: u8 = u8::MAX - 3;
 /// Tag for the CIS2 TokenMetadata event.
 pub const TOKEN_METADATA_EVENT_TAG: u8 = u8::MAX - 4;
 
-/// Sha256 digest
-pub type Sha256 = [u8; 32];
-
-/// The location of the metadata and an optional hash of the content.
-// Note: For the serialization to be derived according to the CIS2
-// specification, the order of the fields cannot be changed.
-#[derive(Debug, Serialize, PartialEq, Eq, Clone)]
-pub struct MetadataUrl {
```

Comment:
Please also re-export MetadatUrl from cis2 so as to not break compatibility.

Reply:
Done.